src/library/transform/scan.cpp

```diff
@@ -1,147 +1,64 @@
-/* ************************************************************************
```

Comment:
In addition this scan.cpp will be removed in the next iteration since I know that algorithm is working properly. We will use only headers for scan, reduce_by_key etc. As mentioned before this is just for testing purposes.